maintainers/maintainer-list.nix

```diff
@@ -5532,6 +5532,11 @@
       fingerprint = "62BC E2BD 49DF ECC7 35C7  E153 875F 2BCF 163F 1B29";
     }];
   };
+  dseelp = {
+    name = "dsee";
+    github = "DSeeLP";
+    githubId = "46624152";
```

Comment:
I didn’t catch that earlier, the githubId must be an integer.

Reply:
Fixed